ar_exercise/exercise.rb

```diff
@@ -51,21 +74,44 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+the_den = Apartment.create(address: "12 Main Street", monthly_rent: 750, sqft: 1000, num_beds: 2, num_baths: 1),
+the_dungeon = Apartment.create(address: "13 Main Street", monthly_rent: 850, sqft: 1200, num_beds: 3, num_baths: 2),
+the_lair = Apartment.create(address: "14 Main Street", monthly_rent: 950, sqft: 1400, num_beds: 3, num_baths: 3)
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
+avery = Tenant.create(name: "Avery", age: 26, gender: "Female", apartment_id: 20),
+bill = Tenant.create(name: "Bill", age: 62, gender: "Male", apartment_id: 20),
+candice = Tenant.create(name: "Candice", age: 54, gender: "Female", apartment_id: 20),
+dillan = Tenant.create(name: "Dillan", age: 33, gender: "Male", apartment_id: 21),
+everett = Tenant.create(name: "Everett", age: 38, gender: "Male", apartment_id: 21),
+fritz = Tenant.create(name: "Fritz", age: 40, gender: "Male", apartment_id: 21),
+georgia = Tenant.create(name: "Georgia", age: 35, gender: "Female", apartment_id: 22),
+happy = Tenant.create(name: "Happy", age: 32, gender: "Female", apartment_id: 22),
+ivan = Tenant.create(name: "Ivan", age: 28, gender: "Male", apartment_id: 22)
 
 # Birthday!
 # It's Kristin Wisoky's birthday. Find her in the DB and change her age to be 1 year older
 # Note: She's in the seed data, so she should be in your DB
+k_wisoky_bday = Tenant.find_by(name: "Kristin Wisoky")
+k_wisoky_bday.increment(:age, 1)
+k_wisoky_bday.save
```

Comment:
i thought increment was only for increasing. In order to set, we can just access kristin's age directly.

```ruby
k_wisoky_bday.age = 1
k_wisoky_bday.save
```